std/haxe/io/Bytes.hx

```diff
@@ -156,7 +171,11 @@ class Bytes {
 		return new Bytes(len, b.slice(pos, pos + len));
 		#end
 	}
-
+	
+	/**
+	  Returns `0` if this instance's bytes and given `other`'s bytes are identical, or a value
+	  different than `0` otherwise.
```

Comment:
Non-zero values also have a meaning, it should be lexicographic comparison (negative value if `this` is shorter or same length + lexicographically comes first, positive if `this` is longer or same length + lexicographically comes later second).